python/requirements_tune.txt

```diff
@@ -1,4 +1,5 @@
-ax-platform
+# ax-platform
+git+https://github.com/facebook/Ax.git@0.1.16
```

Comment:
not sure but do we want to use `git+https://github.com/facebook/Ax.git@0.1.16#egg=ax-platform`?

Reply:
Should work without it, but I updated the PR to make it explicit. Thanks!